otheatre/src/main/java/com/orange/otheatre/otheatre/controller/ManageEventsController.java

```diff
@@ -42,6 +48,11 @@
     @Autowired
     EventService saveEvent;
 
+    @Autowired
+    CheckPlayEligibility eligibility;
```

Comment:
You have EventService, you don't neet another service. Just add functionallity in addEvent method to check if all the prerequisites are satisfied in order to add a new event